core/tests/test_utils.py

```diff
@@ -2323,6 +2323,14 @@ def post_json(
             dict. The JSON response for the request in dict form.
         """
         if use_payload:
+            if 'files' in data:
+                # Bytes type data cannot be used as the json value
+                # json cannot serialize object of type bytes.
+                data['files'] = {
+                    name: utils.convert_image_binary_to_data_str(file)
+                    if isinstance(file, bytes) else file
+                    for name, file in data['files'].items()
+                }
```

Comment:
Do we still need this? Now that the data should be in base64? You should maybe in https://github.com/oppia/oppia/pull/14063/files#diff-738104739b88c1fbd2310135bee0944d2020221de359617cc07e361e3614e096R744 and similar places directly pass the base64 value.

Reply:
Done.